site/source/user-manual/service-guides/nextcloud/nextcloud-setup/nextcloud-ios.rst

```diff
@@ -43,15 +77,23 @@ You will first need to have the :ref:`Tor daemon running<tor-ios>`.
 
 Device Integration
 ------------------
-In order to sync calendars and contacts with your iOS device, follow the steps below, which are adapted from the `Official Nextcloud guide <https://docs.nextcloud.com/server/25/user_manual/en/groupware/sync_ios.html>`_.  First head into the top-righthand menu of your Nextcloud's WebUI and click "Apps," then search for and install the Calendar and/or Contacts Apps.
+In order to sync calendars and contacts with your iOS device, follow the steps below, which are adapted from the `Official Nextcloud guide <https://docs.nextcloud.com/server/25/user_manual/en/groupware/sync_ios.html>`_.  First head into the top-righthand menu of your Nextcloud's WebUI and click "Apps," then search for and install the Calendar and/or Contacts Apps. Next folow this guide for iOS.
```

Comment:
I believe the above 'Tor Setup' section is wrong.  I had copy-pasted the desktop setup to android/ios and I have already fixed the android section.  It references the desktop and I think the directions are wrong (should be much simpler with Orbot) - can you verify and fix please?

Reply:
It sounds like there may be general issues with iOS/Orbot connections right now.  One user suggested that uninstall/re-install of Orbot fixed for him.